modules/openxr/openxr_api.cpp

```diff
@@ -2244,31 +2246,35 @@ bool OpenXRAPI::process() {
 	// As the name suggests, OpenXR can pause the thread to minimize the time between
 	// retrieving tracking data and using that tracking data to render.
 	// OpenXR thus works best if rendering is performed on a separate thread.
-	void *frame_wait_info_next_pointer = nullptr;
-	for (OpenXRExtensionWrapper *extension : frame_info_extensions) {
-		void *np = extension->set_frame_wait_info_and_get_next_pointer(frame_wait_info_next_pointer);
-		if (np != nullptr) {
-			frame_wait_info_next_pointer = np;
-		}
-	}
-
-	XrFrameWaitInfo frame_wait_info = { XR_TYPE_FRAME_WAIT_INFO, frame_wait_info_next_pointer };
-	frame_state.predictedDisplayTime = 0;
-	frame_state.predictedDisplayPeriod = 0;
-	frame_state.shouldRender = false;
+	{
+		GodotProfileZone("OpenXR: xrWaitFrame");
```

Comment:
Since this runs until the end of the function, there's no need to wrap it with braces.

Actually, the pattern `profiling.h` introduces tries to avoid putting stuff in braces in general, since that was a little too intrusive for my taste. In most cases where braces are used, grouped profile zones are preferable.

Reply:
> and it showed something that surprised me with multithreaded rendering

FYI, see [this comment](https://github.com/godotengine/godot/pull/112506#issuecomment-3563754348) for the details about the surprising thing I found with multithreaded rendering